package.json

```diff
@@ -32,6 +32,7 @@
     "bare": ">=1.14.0"
   },
   "devDependencies": {
+    "bare-ffmpeg": "1.0.0-6",
```

Comment:
Let's add a separate `package.json` to the example.

Reply:
yeah make sense